lib/browser_crawler/version.rb

```diff
@@ -1,3 +1,3 @@
 module BrowserCrawler
-  VERSION = "0.1.1"
+  VERSION = "0.1.2"
```

Comment:
there is a lot of changes. I would say the API has changed a lot, hence it might be incompatible. I would recommend to bump up the major version

Reply:
I bumped up the major version.